bar-descriptor.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="utf-8" standalone="no"?>
+<!--
+   Copyright (c) 2011-2015 BlackBerry Limited.
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+   http://www.apache.org/licenses/LICENSE-2.0
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+   -->
+
+<qnx xmlns="http://www.qnx.com/schemas/application/1.0">
+    <id>com.example.Term48_dev</id>
+    <permission>access_shared</permission>
+    <asset path="assets">assets</asset>
```

Comment:
This assets path is bogus? When building the .bar I get an error here on invalid path.